app/form_objects/provider_visa_form.rb

```diff
@@ -0,0 +1,25 @@
+class ProviderVisaForm
+  include ActiveModel::Model
+
+  attr_reader :can_sponsor_skilled_worker_visa, :can_sponsor_student_visa
+
+  validates :can_sponsor_skilled_worker_visa, inclusion: { in: [true, false], message: "Select whether your provider can sponsor skilled worker visas" }
+  validates :can_sponsor_student_visa, inclusion: { in: [true, false], message: "Select whether your provider can sponsor student visas" }
```

Comment:
Copy tweaks (and capitalisation):

```suggestion
  validates :can_sponsor_skilled_worker_visa, inclusion: { in: [true, false], message: "Select if you can sponsor Skilled Worker visas" }
  validates :can_sponsor_student_visa, inclusion: { in: [true, false], message: "Select if you can sponsor Student visas" }
```

Should we move these strings to the locale file?